core/src/actors/connections_manager/actor.rs

```diff
@@ -0,0 +1,19 @@
+use actix::{Actor, Context};
+use log::debug;
+
+use super::ConnectionsManager;
+
+/// Make actor from `ConnectionsManager`
```

Comment:
```suggestion
/// Make actor from ConnectionsManager
```

Reply:
We have not been using ` ... ` elsewhere in comments. Just wanted to check for consistency.